client/src/featureform/list.py

```diff
@@ -0,0 +1,77 @@
+from featureform.proto import metadata_pb2
+from .format import *
+
+def list_name_status(stub, resource_type):
+    stub_list_functions = {
+        "entity": stub.ListEntities,
+        "user": stub.ListUsers
+    }
+    format_rows("NAME", "STATUS")
+    res = []
```

Comment:
does `res` need to be sorted?

Reply:
yea, for the tests, because otherwise, sometimes the resources get printed in a different order

Reply:
You can sort it short hand and even pull it out as separate function. 
```
res = sorted([f for f in stub_list_functions[resource_type](metadata_pb2.Empty())], key=lambda x:x.name)
for f in res:
      format_rows(f.name, f.status.Status._enum_type.values[f.status.status].name)
```

This will remove the need for two lists. Makes it a little cleaner